src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-directories/cephfs-directories.component.html

```diff
@@ -25,9 +25,17 @@
         </cd-table>
 
         <legend i18n>Snapshots</legend>
-        <cd-table [sorts]="snapshot.sortProperties"
-                  [data]="selectedDir.snapshots"
-                  [columns]="snapshot.columns">
+        <cd-table [data]="selectedDir.snapshots"
+                  [columns]="snapshot.columns"
+                  identifier="name"
+                  forceIdentifier="true"
+                  selectionType="multi"
```

Comment:
![Bildschirmfoto vom 2019-11-08 12-43-55](https://user-images.githubusercontent.com/1897962/68474361-764c9f80-0225-11ea-9a2f-7e1a33661000.png)
```suggestion
                  selectionType="multiClick"
```